test/parallel/test-tls-key-mismatch.js

```diff
@@ -29,6 +29,9 @@ if (!common.hasCrypto) {
 const assert = require('assert');
 const tls = require('tls');
 const fs = require('fs');
+const errorMessageRegex = new RegExp('^Error: error:0B080074:x509 ' +
```

Comment:
@sam-github The error message is too long to fit within 80 cols. I'll see if I can condense it to two lines instead of three & then update the PR accordingly.